### DISCUSSION TITLE
Remove Downloaded Model on Windows 11

### DISCUSSION THREAD

**Discussion by Rishabhjain486** [openai/whisper#1654] [Q&A] [answered]
Hello!
I downloaded "large" and "large-v2" models but my laptop is not able to run it.

When I run the command "whisper audio.mp3 --model large-v2" it says "torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 26.00 MiB (GPU 0; 6.00 GiB total capacity; 5.05 GiB already allocated; 0 bytes free; 5.34 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF"

I would like to know how to remove the "large" models already downloaded.

I'm using Windows 11
Thanks.

**Comment by aquorio15** [comment]
I think the problem is happening because of GPU memory not because of the model downloaded

**Reply by Rishabhjain486** [reply]
yes, it is due to the lack of GPU memory, The Large model requires 10GB VRAM which is beyond the capability of my current GPU, hence I was looking for a way to remove/ uninstall the large model to free up the 2.85GB space it took. Would you be able to guide me as to how I may be able to do that?

**Comment by phineas-pta** [comment, marked as answer]
in folder `%USERPROFILE%\.cache`

**Reply by Rishabhjain486** [reply]
Thank you for replying! I am, however, not very well versed as to how I may be able to reach that folder. Would it be possible to give a complete path to reach the file and instructions on how I can subsequently delete/ uninstall the model?

**Reply by phineas-pta** [reply]
copy that path into address bar in explorer

**Reply by Rishabhjain486** [reply]
Thank you so much!
